common/desktop-exports

```diff
@@ -144,3 +144,9 @@ if [ ! -f $DCONF_DEST_USER_DIR/user ]; then
     ln -s /home/$USER/.config/dconf/user $DCONF_DEST_USER_DIR
   fi
 fi
+
+# Testability support
```

Comment:
Can you make that conditional to testability/ directory being available?

It's a Qt only solution, correct, shouldn't it be better to be in the Qt launcher (and I guess only for Qt5 flavors)